packages/common/container-definitions/src/runtime.ts

```diff
@@ -123,7 +123,9 @@ export interface IBatchMessage {
  * @alpha
  */
 export interface IContainerContext {
-	readonly options: ILoaderOptions;
+	// Used to be ILoaderOptions, this is staging for eventual removal.
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	readonly options: Record<string | number, any>;
```

Comment:
Especially now that we've lost type information here, let's document what these options are for.